src/share/util/shr_flux_mod.F90

```diff
@@ -355,6 +358,9 @@ real(R8)    :: ugust  ! function: gustiness as a function of convective rainfall
            tstar = rh * delt
            qstar = re * delq
         enddo
+        if (iter < 1) then
+           call shr_sys_abort('No iterations performed ' //  // errMsg(sourcefile, __LINE__))
```

Comment:
Sorry, my comment wasn't explicit enough here. For this to work, you need to define sourcefile at the top of the file:

```fortran
  character(len=*), parameter, private :: sourcefile = &
       __FILE__
```

and also add a `use` statement:

```fortran
  use shr_log_mod  , only : errMsg => shr_log_errMsg
```
  
Also, I see I had a typo here: there are two instances of `//`